recipe/meta.yaml

```diff
@@ -1,6 +1,7 @@
 {% set version = "5.0.1" %}
 {% set major = version.rpartition('.')[0] %}
-{% set build = 0 %}
+{% set cuda_major = (cuda_compiler_version|default("11.8")).rpartition('.')[0] %}
```

Comment:
Short-cut determine the CUDA major version. Needed a fallback during `conda-build`'s rendering passes (hence `default`). Though the `default` is overridden in the final result